apps/content/api/portal/font_versions.py

```
@@ -0,0 +1,210 @@
+    response={
+        201: FontVersionListOut,
+        400: NinjaErrorResponse[Literal["asset_id_mismatch"]],
+        404: NinjaErrorResponse[Literal["font_not_found"]],
+    },
+)
+@permission_required([permission_class(PermissionChoice.PORTAL_CREATE_FONT)])
+def create_font_version(
+    request: Request,
+    font_slug: str,
+    data: Form[FontVersionCreateIn],
+    file: UploadedFile = File(...),
```

Ruff (0.15.20)

[warning] Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)